lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -123,6 +124,9 @@ def submit(actions)
 
         if SUCCESS_CODES.include?(status)
           next
+	elsif CONFLICT_CODE == status && action[1][:version_type] == "external"
+	  # When using external versioning, conflicts are basically successes.
```

Comment:
I'd like to see `action[1]` pulled out into a more semantically named variable. Something like `action_metadata`. Maybe you have a better idea? I don't love that name.